src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsManager.cs

```diff
@@ -96,11 +96,7 @@ private static void AddBuildInPostProcessor(List<PostProcessor> processorList)
                     ContractName = "html",
                     Processor = new HtmlPostProcessor
                     {
-                        Handlers =
-                        {
-                            new ValidateBookmark(),
-                            new RemoveDebugInfo(),
-                        },
+                      
```

Comment:
Can simplify this to `Processor = new HtmlPostProcessor()`

Reply:
Got it~